x-pack/plugins/enterprise_search/common/connectors/native_connectors.ts

```diff
@@ -1626,6 +1626,32 @@ export const NATIVE_CONNECTOR_DEFINITIONS: Record<string, NativeConnector | unde
         validations: [],
         value: false,
       },
+      use_document_level_security: {
+        default_value: false,
+        depends_on: [],
+        display: DisplayType.TOGGLE,
+        label: i18n.translate(
+          'xpack.enterpriseSearch.nativeConnectors.sharepoint_online.configuration.useDocumentLevelSecurityLabel',
+          {
+            defaultMessage: 'Use document level security',
+          }
+        ),
+        options: [],
+        order: 7,
+        required: true,
+        sensitive: false,
+        tooltip: i18n.translate(
+          'xpack.enterpriseSearch.nativeConnectors.sharepoint_online.configuration.useDocumentLevelSecurityTooltip',
+          {
+            defaultMessage:
+              'This will add access control data to content documents and will also sync identities with their corresponding access control to a separate index .search-acl-filter-index-name',
```

Comment:
```suggestion
              'Document level security ensures identities and permissions set in Sharepoint Online are maintained in Elasticsearch. This metadata is added to your Elasticsearch documents, so you can control user and group read-access. Access control syncs ensure this metadata is kept up to date.',
```

Here's a cut. I'm not sure the fact that we create a separate access control index is relevant here. This gets explained in the docs. 🤷‍♂️